src/plugins/bulma.ts

```diff
@@ -53,6 +51,7 @@ export const bulmaConfig: OrugaOptions = {
         iconLeftClass: "is-left",
         iconRightClass: "is-right",
         placeholderClass: "is-empty",
+        rootVariantClass: "is-" as unknown as boolean, // TODO : fix this hack. waiting on https://github.com/oruga-ui/oruga/pull/812 to fix this type.
```

Comment:
FYI, I'll remove these as statements once https://github.com/oruga-ui/oruga/pull/812 is released